app/src/main/java/com/example/tanya/godeltechchallengeandroid/ui/start/StartActivity.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.tanya.godeltechchallengeandroid.ui.start
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import com.example.tanya.godeltechchallengeandroid.R
+import com.example.tanya.godeltechchallengeandroid.ui.next.HomeActivity
+import dagger.android.support.DaggerAppCompatActivity
+import kotlinx.android.synthetic.main.activity_start.*
+import javax.inject.Inject
+
+class StartActivity : DaggerAppCompatActivity(), StartContract.View {
+
+    @Inject lateinit var presenter: StartContract.Presenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_start)
+
+        presenter.bindView(this)
+
+        presenter.loadData()
+    }
+
+    override fun navigateToHomeScreen() {
+        //TODO: implement navigating to Home screen
+        Toast.makeText(this, resources.getText(R.string.loading_done), Toast.LENGTH_SHORT).show()
+        hideProgress()
+
+        startActivity(Intent(this, HomeActivity::class.java))
```

Comment:
Hmmm! You start the home screen but do not kill the current one. Do you think it is correct behavior if user press back and will see the start-start up screen once again?

Also one more tip: you could create a new abstraction like `navigator` and hide  activity transition logic in the new abstraction. For instance: